library/public/image.h

```diff
@@ -43,8 +43,8 @@ class F3D_EXPORT image
   ~image();
   image(const image& img);
   image& operator=(const image& img);
-  image(image&& img);
-  image& operator=(image&& img);
+  image(image&& img) noexcept;
+  image& operator=(image&& img) noexcept;
```

Comment:
why adding it only here ?

Reply:
We should add it to other functions too, but this one is important because it has an impact on the performance, see https://clang.llvm.org/extra/clang-tidy/checks/performance/noexcept-move-constructor.html